libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/couchdb/AttachmentStreamConnector.java

```diff
@@ -106,15 +109,26 @@ public InputStream unsafeGetAttachmentStream(AttachmentContent attachment) throw
         PipedOutputStream out = new PipedOutputStream(in);
 
         new Thread(() -> {
+            Map<String, Integer> fileNameUsageMap = new HashMap<>();
             byte[] buffer = new byte[1024];
             int length;
 
             try(ZipOutputStream zip = new ZipOutputStream(out)){
                 for(AttachmentContent attachment : attachments) {
-                    // TODO: handle attachments with equal name
-                    ZipEntry zipEntry = new ZipEntry(attachment.getFilename());
-                    zip.putNextEntry(zipEntry);
+                    ZipEntry zipEntry;
+
+                    String originalFileName = attachment.getFilename();
+                    if (!fileNameUsageMap.containsKey(originalFileName)) {
+                        fileNameUsageMap.put(originalFileName, 0);
+                        zipEntry = new ZipEntry(originalFileName);
+                    } else {
+                        int count = fileNameUsageMap.get(originalFileName);
+                        count += 1;
+                        fileNameUsageMap.put(originalFileName, count);
+                        zipEntry = getDuplicatedZipEntry(originalFileName, fileNameUsageMap);
```

Comment:
instead of `getDuplicatedZipEntry` I would expect `getDeduplicatedZipEntry`

Reply:
changed method name to `getDeduplicatedZipEntry`